src/javascript/JContent/ContentRoute/ContentLayout/Upload/UploadItem/UploadItem.jsx

```diff
@@ -96,7 +96,7 @@ export class UploadItem extends React.Component {
                     })}
                     doUploadAndStatusUpdate={this.doUploadAndStatusUpdate}
                 />
-                <div className={classes.grow} doUploadAndStatusUpdate={this.doUploadAndStatusUpdate}/>
```

Comment:
Curious why this portion is removed?

Reply:
There is no such attribute on <div> element so it does nothing